e2e/helpers/types.ts

```diff
@@ -0,0 +1,73 @@
+/* USERS */
+export type LoginResponse = {
```

Comment:
Replicated various types from [synapse-types](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/main/packages/synapse-types) (LoginResponse, FileHandleStatus, FileHandle, PaginatedResults) -- is there a way to pull these types from the package directly?

Reply:
Nope because it's private to the monorepo. This is fine for now IMO.

Kind of want to wait for https://sagebionetworks.jira.com/browse/PLFM-7768 to be done to spend more time wrangling types or SynapseClient.

Reply:
Got it, thanks! I've created a ticket to track creating the synapse-types package [here](https://sagebionetworks.jira.com/browse/SWC-6529).